src/crewai/cli/plus_api.py

```diff
@@ -27,6 +27,9 @@ def _make_request(self, method: str, endpoint: str, **kwargs) -> requests.Respon
         url = urljoin(self.base_url, endpoint)
         return requests.request(method, url, headers=self.headers, **kwargs)
 
+    def login_to_tool_repository(self):
```

Comment:
Out of curiosity, is the tool repo login credentials different from crewai-plus' credentials?

Reply:
The tool repo credential is a username and password pair used by Poetry to authenticate with our PyPI repository using HTTP Basic. Some differences:

- Plus creds are short lived and may expire, while tool repo creds never expire
- Plus creds can authenticate to any API endpoint, while tool repo creds only work for downloading tools
- Plus creds are stored in the CLI, while tool repo creds are stored and managed by Poetry

Reply:
Got it! Thanks! :)